### ISSUE TITLE
如何删除通过vue ui界面添加的插件 [vue-cli3.1.3]

### ISSUE THREAD

**Issue by jinweizhiyuan** [vuejs/vue-cli#2962]
### Version
3.1.1

### Reproduction link
[http://localhost:8000/plugins](http://localhost:8000/plugins)

### Node and OS info
8.11.1, npm/window 7

### Steps to reproduce
在vue-cli 界面中选择左侧的插件菜单，进入插件列表，通过搜索添加插件按钮添加element插件，之后项目中用不到了，没找到删除插件的地方

### What is expected?
希望提供删除插件的按钮

### What is actually happening?
实际上找不到删除插件的地方

**Comment by CaiPeng1989** [comment]
你是想要删除 `vue-cli-plugin-element` 还是 `element-ui` ？

**Comment by jinweizhiyuan** [comment]
既然删除的话，应该都删除包括vue-cli-plugin-element 和 element-ui

**Comment by jinweizhiyuan** [comment]
> 你是想要删除 `vue-cli-plugin-element` 还是 `element-ui` ？

既然删除的话，应该都删除包括vue-cli-plugin-element 和 element-ui

**Comment by CaiPeng1989** [comment]
> 既然删除的话，应该都删除包括vue-cli-plugin-element 和 element-ui
但是 `vue-cli-plugin-element` 不只是帮我们安装了 `element-ui`，还修改了 `App.vue`  的内容。

**Comment by Ale-cc** [comment]
是的 UI上面是没有提供删除的操作按钮。只能手动删了，全局搜索插件名，以vue-cli-plugin-scss-base 为例，逐个删吧！
![image](https://user-images.githubusercontent.com/18302674/50880491-833c0f00-1419-11e9-82da-10d40f486472.png)

![image](https://user-images.githubusercontent.com/18302674/50880475-73242f80-1419-11e9-9ff4-b0e5d4187b04.png)
